JSDemos/Demos/Scheduler/CellTemplates/React/App.tsx

```diff
@@ -4,6 +4,7 @@ import React from 'react';
 import Scheduler, { SchedulerTypes } from 'devextreme-react/scheduler';
 import notify from 'devextreme/ui/notify';
 
+import dxForm from 'devextreme/ui/form';
```

Comment:
It's better not to import types from 'devextreme' package
Better to use `Form` type 'devextreme-react' and change function arg to ref
Or use `Form['instance']` type